conf/portal.routes

```diff
@@ -45,5 +45,8 @@ PUT        /v1/expressions                      controllers.ExpressionController
 # Templates
 GET        /assets/html/HeaderViewModel.html    controllers.ApplicationController.getHeaderViewModel
 
+# Kairosdb proxying
+-> /api/v1 kairosdb.Routes
```

Comment:
I assume this for compatibility with existing Kairosdb clients? -- can you add a note for that, because it seems odd otherwise. Ideally, we should get Grafana to include a path prefix for its Kairosdb data source.